cmd/create.go

```diff
@@ -178,7 +184,7 @@ A full list of component types that can be deployed is available using: 'odo cat
 			// Deploy the component with a binary
 
 			// Retrieve the path of the binary
-			path, err := filepath.Abs(componentBinary)
+			path, err := util.GetAbsPath(componentBinary)
```

Comment:
Same here, why are we not using the paths retrieved above?

Reply:
Same as above